grai-server/app/the_guide/settings/base.py

```diff
@@ -326,7 +326,7 @@ def inner(value: str | bool) -> bool:
 GITHUB_APP_ID = config("GITHUB_APP_ID", None)
 GITHUB_PRIVATE_KEY = config("GITHUB_PRIVATE_KEY", None)
 
-NGROK_URL = config("NGROK_URL", f"https://{SERVER_HOST}")
+SERVER_URL = config("NGROK_URL", f"https://{SERVER_HOST}")
```

Comment:
Could we also change the environment variable reference from `NGROK_URL` i.e.

```python
SERVER_URL = config("SERVER_URL", f"https://{SERVER_HOST}")
```

Reply:
yes, I'll fix it.